### PR TITLE
util: Move ResolveErrMsg to util/error

### PR DESCRIPTION
**Pull request by maflcko**
Pull request https://github.com/bitcoin/bitcoin/pull/16248#discussion_r314035862 duplicated the body of this util function. The whole point of the util function is to be shared, so do that here as a fixup to #16248

### CODE REVIEW

**Comment by Sjors** [comment]
utACK fa27c55

File: src/init.cpp [hunk 1]
```diff
@@ -822,11 +822,6 @@ void InitParameterInteraction()
     }
 }
 
-static std::string ResolveErrMsg(const char * const optname, const std::string& strBind)
```
Comment (promag) [inline src/init.cpp]: Shoud `#include <util/error.h>` then?

**Comment by NicolasDorier** [comment]
utACK seems like travis need a kick-in-the-box (and maybe need https://github.com/bitcoin/bitcoin/pull/16620#discussion_r314538392)

**Review by ryanofsky — APPROVED** [review]
utACK fa27c55b0593c769b6ad87de0b59df3816d73548

File: src/init.cpp (hunk 1 above)
Comment (MarcoFalke) [reply src/init.cpp]: Will leave as is for now to not invalidate review

**Pull request merged**